Entities/Arm.cpp

```diff
@@ -648,16 +648,17 @@ void Arm::DrawHand(BITMAP *pTargetBitmap,
                    DrawMode mode) const
 {
 
-    Vector handPos(m_JointPos.GetFloored() +
+    Vector handPos(m_JointPos +
                    m_HandOffset +
                    (m_Recoiled ? m_RecoilOffset : Vector()) -
                    targetPos);
     handPos.m_X -= (m_pHand->w / 2) + 1;
     handPos.m_Y -= (m_pHand->h / 2) + 1;
+
     if (!m_HFlipped)
-        draw_sprite(pTargetBitmap, m_pHand, handPos.m_X, handPos.m_Y);
+        draw_sprite(pTargetBitmap, m_pHand, handPos.GetRoundIntX(), handPos.GetRoundIntY());
```

Comment:
Right now `GetRoundIntX` does `std::floorf(m_X + 0.5)`, how about changing it to `std::round(m_X)` and checking how it affects stuff?
Same for `GetRoundIntY` obviously.

If you don't want to right now then it's ok, everything looks fine to me.

Reply:
i actually did alot of changes to the vector methods in a different branch, and what you're saying is one of them. it works fine.
i can probably already send a PR of it, just need to separate it out from a branch with a bunch of other things.